src/sunlinsol/band/sunlinsol_band.c

```diff
@@ -133,27 +116,23 @@ SUNLinearSolver_ID SUNLinSolGetID_Band(SUNLinearSolver S)
   return (SUNLINEARSOLVER_BAND);
 }
 
-int SUNLinSolInitialize_Band(SUNLinearSolver S)
+SUNErrCode SUNLinSolInitialize_Band(SUNLinearSolver S)
 {
   /* all solver-specific memory has already been allocated */
-  LASTFLAG(S) = SUNLS_SUCCESS;
-  return (SUNLS_SUCCESS);
+  LASTFLAG(S) = SUN_SUCCESS;
+  return SUN_SUCCESS;
 }
 
 int SUNLinSolSetup_Band(SUNLinearSolver S, SUNMatrix A)
 {
+  /* Error checks in this function must be NoRet because the return value
+     is an integer code specific to the SUNLinearSolver. */
```

Comment:
I guess what I meant is that if `SUNErrCode` was not a `typedef` to an `int` than the `NoRet` macros would need to be used. However, since the semantics of the return value (negative value for unrecoverable, positive for recoverable) are compatible with the semantics of `SUNErrCode` then I suppose it is not a problem.